packages/vue-instantsearch/src/components/Highlight.js

```diff
@@ -0,0 +1,44 @@
+import { createHighlightComponent } from 'instantsearch-ui-components';
+
+import { createSuitMixin } from '../mixins/suit';
+import { renderCompat } from '../util/vue-compat';
+
+export default {
+  name: 'AisHighlight',
+  mixins: [createSuitMixin({ name: 'Highlight' })],
+  // props: {
```

Comment:
is this file meant to be committed already?

Reply:
Nope, I committed it by accident. Removed!